src/components/gallery/GalleryHeader.tsx

```diff
@@ -0,0 +1,9 @@
+const GalleryHeader = () => {
+  return (
+    <div className="font-banco flex w-full flex-row justify-center text-left text-[108px] font-medium leading-[103.25px]">
```

Comment:
Instead of text-108px , can you use established sizes 

i.e. => text-3xl , text-2xl etc

Reply:
Alrighty! I changed it to `text-8xl` bc it seems to be the closest font text size as text-108px